astropy/io/ascii/tests/test_cds_header_from_readme.py

```diff
@@ -34,22 +33,20 @@ def test_description():
     data = "data/cds/description/table.dat"
     for read_table in (read_table1, read_table2, read_table3):
         table = read_table(readme, data)
-        assert_equal(len(table), 2)
-        assert_equal(table["Cluster"].description, "Cluster name")
-        assert_equal(table["Star"].description, "")
-        assert_equal(table["Wave"].description, "wave ? Wavelength in Angstroms")
-        assert_equal(table["El"].description, "a")
-        assert_equal(
-            table["ion"].description, "- Ionization stage (1 for neutral element)"
-        )
-        assert_equal(
-            table["loggf"].description,
+        assert len(table) == 2
+        assert table["Cluster"].description == "Cluster name"
+        assert table["Star"].description == ""
+        assert table["Wave"].description == "wave ? Wavelength in Angstroms"
+        assert table["El"].description == "a"
+        assert table["ion"].description == "- Ionization stage (1 for neutral element)"
+        assert (
+            table["loggf"].description ==
             "log10 of the gf value - logarithm base 10 of stat. weight times "
             "oscillator strength",
         )
```

Comment:
Ruff correctly detects an [F631 (assert-tuple)](https://docs.astral.sh/ruff/rules/assert-tuple/) violation here. [Our developer documentation recommends using the `pre-commit` tool](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit), which would automatically detect such mistakes before a commit was even made.
```suggestion
        assert (
            table["loggf"].description ==
            "log10 of the gf value - logarithm base 10 of stat. weight times "
            "oscillator strength"
        )
```